docs/source/datasets/index.rst

```diff
@@ -461,4 +461,148 @@ The following columns are included in these datasets:
 Tax form: 1040
 --------------
 As with data collected from W-2 and 1099 forms, Pseudopeople will also enable the simulation of administrative records from 1040 forms, which are
-also reported to the IRS on an annual basis. This feature has not yet been implemented, so please stay tuned for more information! 
+also reported to the IRS on an annual basis. To find out more about the 1040 tax form, visit the `IRS information page <https://www.irs.gov/instructions/i1040gi>`_.
+
+Generate 1040 data with pseudopeople.generate_taxes_1040.
+
+The following columns are included in these datasets:
+
+.. list-table:: **Dataset columns**
+   :header-rows: 1
+
+   * - Attribute Name
+     - Column Name
+     - Notes
+   * - Unique simulant ID
+     - :code:`simulant_id`
+     - Not affected by noise functions; intended use is "ground truth" for testing and validation.
+   * - Unique household ID
+     - :code:`household_id`
+     - Not affected by noise functions; intended use is "ground truth" for testing and validation; consistent across all
+       datasets. 
+   * - First name
+     - :code:`first_name`
+     - 
+   * - Middle initial
+     - :code:`middle_initial`
+     - 
+   * - Last name
+     - :code:`last_name`
+     - 
+   * - Age
+     - :code:`age`  
+     - Rounded down to an integer.
+   * - Date of birth
+     - :code:`date_of_birth`
+     - Formatted as YYYY-MM-DD.
+   * - Mailing address street number
+     - :code:`mailing_address_street_number`
+     - 
+   * - Mailing address street name
+     - :code:`mailing_address_street_name`
+     - 
+   * - Mailing address unit number
+     - :code:`mailing_address_unit_number`
+     - 
+   * - Mailing address PO box
+     - :code:`mailing_address_po_box`
+     - 
+   * - Mailing address city
+     - :code:`mailing_address_city`    
+     - 
+   * - Mailing address state
+     - :code:`mailing_address_state`  
+     - 
+   * - Mailing address ZIP code
+     - :code:`mailing_address_zipcode`
+     - 
+   * - Social security number 
+     - :code:`ssn`
+     - 
+   * - Individual tax identification number
```

Comment:
According to [Zeb](https://ihme.slack.com/archives/C02KUQ9LX32/p1688762729760379?thread_ts=1688762551.503559&cid=C02KUQ9LX32) and to [the 1040 form itself](https://www.irs.gov/pub/irs-pdf/f1040.pdf), we shouldn't have a separate column for ITIN. Instead ITIN's should go in the SSN column, and we should add a note to that column saying that values in that column will be ITIN's for people who use those instead of SSN's.